ext/dom/tests/modern/html/parser/parser_warning_03.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE HTML>
+<html>
+    <body>
+        <p></>
+    </body>
+</html>
```

Comment:
I'm guessing the missing EOL before EOF are on purpose for the html files?

Reply:
Yes. Text content (including whitespace) outside of the html document, will be moved to the body. So it does actually make a difference when working with HTML documents. It does not matter however for browsers because they only care about the rendering.